palak/src/popup/Popup.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from "react";
+import { MdOutlineCancel } from "react-icons/md";
+
+interface Colors {
+  bgColor: string;
+  glassBgColor: string;
+  textColor: string;
+  inputBgColor: string;
+  buttonBgColor: string;
+  spanColor: string;
+  popupBgColor: string;
+  popupButton: string;
+  inputBorder: string;
+}
+
+const Popup = ({
+  bgColor,
+  glassBgColor,
+  textColor,
+  inputBgColor,
+  buttonBgColor,
+  spanColor,
+  popupBgColor,
+  popupButton,
+  inputBorder,
+}: Colors) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div
+      className={`relative inset-0 h-screen w-screen flex items-center justify-center ${bgColor}`}
+      onMouseDownCapture={() => setIsOpen(false)}
+    >
+      <div className={`absolute inset-0 transition-all duration-300 ${isOpen ? "bg-black/60" : ""}`}></div>
+
+      {/* GlassEffect Popup */}
+      <div
+        className={`absolute z-20 w-96 max-h-screen pb-20 flex flex-col rounded-md shadow-lg transform transition-all duration-300 ${glassBgColor} backdrop-blur-sm`}
+      >
+        <p className={`mt-10 mx-auto ${textColor} font-semibold text-lg`}>
+          Login for <span className={`${spanColor}`}>better</span> experience
+        </p>
+        <label className={`mt-12 px-8 ${textColor}`}>Email</label>
+        <input
+          type="text"
+          placeholder="Your Email here.."
+          className={`mx-auto mt-1 rounded-md h-10 w-80 focus-visible:outline-none focus-visible:ring-1 ${inputBorder} p-2 text-sm ${inputBgColor} backdrop-blur-md placeholder:text-gray-400`}
+        />
+        <label className={`mt-6 px-8 ${textColor}`}>Password</label>
+        <input
+          type="text"
+          placeholder="Your Password here.."
+          className={`mx-auto mt-1 rounded-md h-10 w-80 focus-visible:outline-none focus-visible:ring-1 focus:ring-blue-400 p-2 border border-blue-300 text-sm ${inputBgColor} ${inputBorder} backdrop-blur-md`}
+        />
+        <a className={`text-sm px-8 ${textColor} underline mt-1.5`} href="">
```

Comment:
if the a tag is not doing anything, the value of href should be `#` ie. `href="#"`